charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml

```diff
@@ -10,6 +10,7 @@ spec:
   names:
     categories:
     - karpenter
+    - nap
```

Comment:
have customers asked for this?

Reply:
No, but they will be delighted it is there